src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/fuse_reshape_transpose_to_sdpa.cpp

```diff
@@ -34,212 +34,212 @@ namespace test {
+//             *first++ = value;
+//             value += stride;
+//         }
+//     }
+//     void generate(int idx, const std::vector<ov::Shape>& targetInputStaticShapes) {
+//         inputs.clear();
+//         auto create_input = [this] (std::shared_ptr<ov::op::v0::Parameter> param, ov::Shape shape, float val) {
+//             if (param->get_element_type() == ov::element::i32) {
+//                 ov::Tensor t{ov::element::i32, shape};
+//                 auto size = ov::shape_size<ov::Shape>(shape);
+//                 auto* p = static_cast<int*>(t.data());
+//                 auto start = static_cast<int>(val);
+//                 for (size_t i = 0; i < size; i++) {
+//                     p[i] = (start + i) % size;
+//                 }
+//                 inputs.insert({param, t});
+//             } else if (param->get_element_type() == ov::element::f32) {
+//                 ov::Tensor t{ov::element::f32, shape};
+//                 strided_iota(static_cast<float*>(t.data()), t.get_size(), val, 0.1f);
+//                 inputs.insert({param, t});
+//             } else {
+//                 ASSERT_TRUE(param->get_element_type() == ov::element::bf16);
+//                 ov::Tensor t{ov::element::bf16, shape};
+//                 strided_iota(static_cast<ov::bfloat16*>(t.data()), t.get_size(), val, 0.1f);
+//                 inputs.insert({param, t});
+//             }
+//         };
+//         // q, k, v
+//         create_input(function->get_parameters()[0], targetInputStaticShapes[0], idx + 1.0f);
+//         create_input(function->get_parameters()[1], targetInputStaticShapes[0], idx + 2.0f);
+//         create_input(function->get_parameters()[2], targetInputStaticShapes[0], idx + 3.0f);
+//     }
+//     void prepare() {
+//         compile_model();
+//         inferRequest = compiledModel.create_infer_request();
+//         ASSERT_TRUE(inferRequest);
+//     }
+//     void reset() {
+//         for (auto&& state : inferRequest.query_state()) {
+//             state.reset();
+//         }
+//     }
+
+//     std::vector<ov::Tensor> run_test(std::shared_ptr<ov::Model> model) {
+//         function = model;
+//         prepare();
+//         std::vector<ov::Tensor> outputs;
+//         int idx = 0;
+//         for (auto&& shapes : targetStaticShapes) {
+//             generate(idx++, shapes);
+//             for (const auto& input : inputs) {
+//                 inferRequest.set_tensor(input.first, input.second);
+//             }
+//             inferRequest.infer();
+//             auto outputTensor = inferRequest.get_output_tensor(0);
+//             ov::Tensor copy{outputTensor.get_element_type(), outputTensor.get_shape()};
+//             outputTensor.copy_to(copy);
+//             outputs.push_back(copy);
+//             reset();
+//         }
+//         return outputs;
+//     }
+// };
+
+// TEST_P(FuseSDPAReshapeTransposeTest, CompareWithRefs) {
+//     SKIP_IF_CURRENT_TEST_IS_DISABLED();
```

Comment:
maybe add `GTEST_SKIP() << "TODO: investigate perf-regression on ICX." << std::endl;` instead of commenting all file?

Reply:
Done.